### PR TITLE
removes the debug check box in button edit overlay

### PR DESCRIPTION
**Pull request by RaphaelAlvez**
adresses #854

Not much about it. Just removed the checkbox in the HTML and the corresponding JS in editmode.js

### CODE REVIEW

**Comment by ArnoldSmith86** [comment]
PR-SERVER-BOT: You can play around with it here: http://212.47.248.129:3899/pr-test (or any other room on that server)

**Comment by rogerl50** [comment]
You should also remove the `#debugButtonOverlay` in `layout.css`, and you may as well take the (unused) debug parameter out of `setText` and its callers in `widget.js`.

**Comment by RaphaelAlvez** [comment]
> You should also remove the `#debugButtonOverlay` in `layout.css`, and you may as well take the (unused) debug parameter out of `setText` and its callers in `widget.js`.

Done. None of these were related to what I was doing but it's good to clean the code a bit.

That CSS was for the button in the debug overlay

**Comment by rogerl50** [comment]
> > You should also remove the `#debugButtonOverlay` in `layout.css`, and you may as well take the (unused) debug parameter out of `setText` and its callers in `widget.js`.
> 
> Done. None of these were related to what I was doing but it's good to clean the code a bit.
> 
> That CSS was for the button in the debug overlay

Yeah, I know, but we may as well get rid of all the `debug` traces at once...

**Comment by RaphaelAlvez** [comment]
> > > You should also remove the `#debugButtonOverlay` in `layout.css`, and you may as well take the (unused) debug parameter out of `setText` and its callers in `widget.js`.
> > 
> > 
> > Done. None of these were related to what I was doing but it's good to clean the code a bit.
> > That CSS was for the button in the debug overlay
> 
> Yeah, I know, but we may as well get rid of all the `debug` traces at once...

I agree. no need for another PR

File: client/js/widgets/widget.js [hunk 1]
```diff
@@ -1567,7 +1567,7 @@ export class Widget extends StateManaged {
     await super.setPosition(x, y, z);
   }
 
-  async setText(text, mode, debug, problems) {
+  async setText(text, mode, problems) {
```
Comment (rogerl50) [inline client/js/widgets/widget.js:1570]: There are two calls to setText that also need to have this parameter removed.

**Review by rogerl50 — CHANGES_REQUESTED** [review]

**Review by rogerl50 — APPROVED** [review]

**Pull request merged**